mysql-test/t/table_open_cache_triggers_functionality.test

```diff
@@ -0,0 +1,216 @@
+--echo #
+--echo # Test coverage for limit on number of table objects in Table Cache
+--echo # with fully-loaded triggers.
+--echo #
+
+--echo
+--echo # Create helper verify_status_increment() routine to be able to
+--echo # to check easily how values of Table Cache and Table Cache triggers
+--echo # hits/misses/overflows status variables have changed.
+--disable_query_log
+DELIMITER |;
+CREATE PROCEDURE
+verify_status_increment(tc_misses_inc INT, trg_hits_inc INT, trg_misses_inc INT, trg_overflows_inc INT)
```

Comment:
May be rework this with MTR `include/assert.inc`s.
Relying on recorded values in the `.result` file is not as safe. People tend to simply re-record `.result` files and such re-records in the `.result` files are very often skipped by reviewers.
Something like,
```
--let $expected_tc_misses = 0
--let $expected_trg_hits = 0
--let $expected_trg_misses = 0
--let $expected_trg_overflows = 0
--source include/assert_cache_misses.inc
```
or even
```
--let $expected_cache_misses_pattern = "0100"
--source include/assert_cache_misses.inc
```
Inside `assert_cache_misses.inc` you would do use `--source include/assert.inc` to check if values match.

Reply:
Changed code along this suggestion.